### ISSUE TITLE
Colormaps Docs Error

### ISSUE THREAD

**Issue by pratiman-91** [lukelbd/proplot#301]
### Description

The latest docs have error in Colormaps section https://proplot.readthedocs.io/en/latest/colormaps.html#

### Steps to reproduce

A "[Minimal, Complete and Verifiable Example](http://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports)" will make it much easier for maintainers to help you.

```python
import proplot as pplt
fig, axs = pplt.show_cmaps(rasterize=True)
```

**Expected behavior**: [What you expected to happen]
List of included colormaps

**Actual behavior**: [What actually happened]
```TypeError: '>=' not supported between instances of 'list' and 'float'```

**Comment by lukelbd** [comment]
Yikes 😬 -- thanks for the heads up. Looking into this.

**Comment by lukelbd** [comment]
It seems the latest proplot version is triggering an internal matplotlib error with [`_ColorbarAutoMinorLocator`](https://github.com/matplotlib/matplotlib/blob/53ba85e38fbf6cce030befa60e0e86b13b278087/lib/matplotlib/colorbar.py#L277-L282). When minor ticks are "active" and the major locator has no ticks (e.g. because it is a `NullLocator` or empty `FixedLocator`) then [`AutoMinorLocator`](https://github.com/matplotlib/matplotlib/blob/919145fe9849c999aa491457c6de6faede5959c3/lib/matplotlib/ticker.py#L2919-L2934) returns an empty list `[]`, while `_ColorbarAutoMinorLocator` always expects numpy arrays.

 This is fixed in the latest version (55e2efa). I came across this matplotlib bug before, but it looks like matplotlib's master branch includes a refactoring of colorbar ticking internals that fixes it -- so we don't need to submit a PR.
